helm/resource_release.go

```diff
@@ -349,6 +349,8 @@ func resourceReleaseCreate(d *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 
+	info(res.GetRelease().GetInfo().GetStatus().GetNotes())
```

Comment:
I believe this should be done in the `resourceReleaseRead()` function, and then `resourceReleaseCreate()` and `resourceReleaseUpdate()` should invoke `resourceReleaseRead()` on their last return statement.

You may have a look at this as an example of this pattern:
- https://github.com/terraform-providers/terraform-provider-kubernetes/blob/master/kubernetes/resource_kubernetes_service.go#L197
- https://github.com/terraform-providers/terraform-provider-kubernetes/blob/master/kubernetes/resource_kubernetes_service.go#L267

Reply:
On second thought, the issue may rather be that it takes a while until the [release status](https://github.com/helm/helm/blob/master/docs/helm/helm_status.md) is [DEPLOYED](https://github.com/helm/helm/blob/master/_proto/hapi/release/status.proto#L31) at which point the notes are actually set.